src/jupyter_contrib_nbextensions/nbextensions/hinterland/hinterland.yaml

```diff
@@ -2,5 +2,38 @@ Type: Jupyter Notebook Extension
 Main: hinterland.js
 Name: Hinterland
 Link: README.md
-Description: Enable code autocompletion menu for every keypress in a code cell, instead of only enabling it with tab
+Description: |
+  Enable code autocompletion menu for every keypress in a code cell, instead of
+  only enabling it with tab
 Compatibility: 4.x
+Parameters:
+- name: enable_at_start
```

Comment:
hinterland prefix missing for parameters (since the hinterland section is read in the js source)